### PR TITLE
remove lint ignore W0613

### PR DESCRIPTION
**Pull request by Nonj**
* Remove W0613 from lint ignore in setup and strict
  * change files to comply with linter

Connected with ticket #480 and PR #1333 

Signed-off-by: Nattanon Bunyatipanon <Nattanon.Bunyatipanon1@T-Mobile.com>

### CODE REVIEW

File: rbac/server/api/errors.py [hunk 1]
```diff
@@ -96,15 +98,16 @@ class ApiInternalError(ApiException):
 
 @ERRORS_BP.exception(ApiException)
 def api_json_error(request, exception):
+    if not request:
```
Comment (nadiabahrami) [inline rbac/server/api/errors.py]: Why did you user two different methods of handling this here? at line 101 and at line 110? in one you remove it.  and in the other you  keep and add a logger statement?

File: rbac/server/api/roles.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
 from rbac.common import rbac
 
-from rbac.server.api.errors import ApiNotImplemented
+# from rbac.server.api.errors import ApiNotImplemented
```
Comment (nadiabahrami) [inline rbac/server/api/roles.py]: Please delete all commented out lines in this file having to do with API not implemented.  If they are necessary in the future we will implement from scratch

File: rbac/server/api/swagger.py [hunk 3]
```diff
@@ -16,11 +16,17 @@
...
+
+LOGGER = get_default_logger(__name__)
+
 SWAGGER_BP = Blueprint("swagger")
 
 
 @SWAGGER_BP.get("api/swagger")
 async def get_swagger(request):
+    if not request:
+        LOGGER.debug(str(request))
```
Comment (nadiabahrami) [inline rbac/server/api/swagger.py]: Does removing request not work here?

File: rbac/server/api/tasks.py [hunk 4]
```diff
@@ -20,7 +20,7 @@
 
 from rbac.common import rbac
 
-from rbac.server.api.errors import ApiNotImplemented
+# from rbac.server.api.errors import ApiNotImplemented
```
Comment (nadiabahrami) [inline rbac/server/api/tasks.py]: Same in this file.  Please remove all commented out endpoints from ApiNotImplemented

File: rbac/server/api/errors.py (hunk 1 above)
Comment (Nonj) [reply rbac/server/api/errors.py]: Woops! Sorry, a couple of methods were throwing errors during tests when I removed request, so I decided to add request back as an argument but I missed this one

File: rbac/server/api/roles.py (hunk 2 above)
Comment (Nonj) [reply rbac/server/api/roles.py]: Gotcha!

File: rbac/server/api/swagger.py (hunk 3 above)
Comment (Nonj) [reply rbac/server/api/swagger.py]: It does not because request is still not being used anywhere in the method

Comment (nadiabahrami) [reply rbac/server/api/swagger.py]: right so can you not remove request from the parameters?  Or is it required?

Comment (Nonj) [reply rbac/server/api/swagger.py]: It is required, unfortunately.

File: rbac/server/api/users.py [hunk 5]
```diff
@@ -23,7 +23,7 @@
 from rbac.common.crypto.keys import Key
 from rbac.common.crypto.secrets import encrypt_private_key
 
-from rbac.server.api.errors import ApiNotImplemented
+# from rbac.server.api.errors import ApiNotImplemented
```
Comment (mtn217) [inline rbac/server/api/users.py]: Should remove commented out code if it is not needed.

File: rbac/server/api/users.py [hunk 6]
```diff
@@ -197,17 +189,15 @@
...
+    user_resource = await users_query.fetch_user_relationships(conn, user_id)
     conn.close()
     return await utils.create_response(conn, request.url, user_resource, head_block)
 
 
-@USERS_BP.patch("api/users/<user_id>")
-@authorized()
-async def update_user(request, user_id):
-    raise ApiNotImplemented()
+# @USERS_BP.patch("api/users/<user_id>")
```
Comment (mtn217) [inline rbac/server/api/users.py]: Should remove commented out code if it is not needed.

File: rbac/server/api/users.py [hunk 7]
```diff
@@ -377,7 +353,7 @@
 
 @USERS_BP.get("api/users/<user_id>/roles/recommended")
 @authorized()
-async def fetch_recommended_roles(request, user_id):
+async def fetch_recommended_roles(request):
```
Comment (mtn217) [inline rbac/server/api/users.py]: When I run the app and I log in I get this error:
fetch_recommended_roles() got an expected keyword argument 'user_id'. Looks like Phil is passing in the user_id variable into this role. Possibly work with Phil to find out what the user_id is used for and if he could change the front end to not pass in the user_id that should fix this error.

Comment (Nonj) [reply rbac/server/api/users.py]: Spoke with Phil, he just put out a PR #1333 to remove code calling this unused endpoint

Comment (mtn217) [reply rbac/server/api/users.py]: It appears PR #1333 have been closed as Phil needs to make more changes. I'll check this PR again once he has merged his changes on the UI side.

**Pull request merged**